src/db/migrator/helper.clj

```diff
@@ -109,7 +109,7 @@
         "add constraint"
         (or (helper/sqlize fk-name) (str from "_" to "_fk"))
         "foreign key"
-        (str "(" (or (helper/sqlize (str col "_id") to) ")"))
+        (str "(" (or (helper/sqlize (str col "_id")) to) ")")
```

Comment:
Actually, now that I think about it, this is in the wrong order:
`(or to (helper/sqlize (str col "_id")))`
If the user doesn't pass a `to` then the helper should kick in.